<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGui::EditorAutoCorrectionOptionPage</class>
 <widget class="QWidget" name="QtGui::EditorAutoCorrectionOptionPage">
  <property name="windowTitle">
   <string>Auto correction</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox { font-weight: bold };</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Auto correction/completion will be applied when showing the selected tag fields after &lt;span style=&quot; font-style:italic;&quot;&gt;loading&lt;/span&gt; a file but &lt;span style=&quot; font-style:italic;&quot;&gt;not&lt;/span&gt; before saving. This means the tag values you see when opening a file are already corrected. This also means when clicking the &amp;quot;Save&amp;quot; button the tags are saved as they are shown in the editor at this point with no further correction applied.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="featuresGroupBox">
     <property name="title">
      <string>Auto correction/completion features to be used</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="insertTitleFromFilenameCheckBox">
        <property name="text">
         <string>Insert title from filename if none present</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="trimWhitespacesCheckBox">
        <property name="text">
         <string>Trim whitespaces</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="formatNamesCheckBox">
        <property name="text">
         <string>Format names like &quot;an example_name&quot; to &quot;An Example Name&quot;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="fixUmlautsCheckBox">
        <property name="text">
         <string>Fix umlauts (replaces ae with ä for example)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="customSubstitutionGroupBox">
     <property name="title">
      <string>Custom substitution</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="matchLabel">
        <property name="text">
         <string>Match pattern</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="replacementLabel">
        <property name="text">
         <string>Replacement</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="regularExpressionLineEdit">
        <property name="placeholderText">
         <string>regular expression</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="replacementLineEdit">
        <property name="placeholderText">
         <string>replacement (use \1, \2, … for capture expressions)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="fieldsLabel">
     <property name="styleSheet">
      <string notr="true">font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Fields to apply the auto correction on</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="fieldsListView">
     <property name="showDropIndicator" stdset="0">
      <bool>true</bool>
     </property>
     <property name="flow">
      <enum>QListView::LeftToRight</enum>
     </property>
     <property name="isWrapping" stdset="0">
      <bool>true</bool>
     </property>
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
